core/spec/models/spree/order_contents_spec.rb

```diff
@@ -307,7 +307,13 @@
   end
 
   context "completed order" do
-    let(:order) { Spree::Order.create! state: 'complete', completed_at: Time.current }
+    let(:order) do
+      Spree::Order.create!(
```

Comment:
Using a factory would have avoided needing to fix this. Maybe we can start using a factory now?

Reply:
Hey! Good to hear from you :) 

This spec tests something hard to test with a factory if I remember correctly. Let's not do this in this PR.